blocks/bricks/conv.py

```diff
@@ -207,16 +220,105 @@ def apply(self, input_):
             with the last two dimensions downsampled.
 
         """
-        output = max_pool_2d(input_, self.pooling_size, st=self.step)
+        output = max_pool_2d(input_, self.pooling_size, st=self.step,
```

Comment:
I think it's less necessary to make it hackable since all options are currently available with the basic implementation, which is not the case for convolution (but will be when we move to the abstract_conv2d stuff). However it is important to have good defaults and a big fat warning about `ignore_border`.